extensions/wikia/Email/Controller/CommentController.class.php

```diff
@@ -104,7 +104,7 @@ protected function getSalutation() {
 	}
 
 	protected function getSummary() {
-		$articleTitle = $this->title->getText();
+		$articleTitle = $this->pageTtitle->getText();
 
 		return wfMessage( $this->getSummaryKey(), $articleTitle )
 			->inLanguage( $this->targetLang )
```

Comment:
Ok, I can't comment on this line because its not in the diff, but a few lines below on L117 we call ```newFromTitle```.  This ends up calling (via hooks) ```ArticleCommentList::ArticleFromTitle```.  This sets the fragment ```#comm-XXX``` and _somehow_ this persists and is applied to the next Title object created.  I'm not positive but I think its because it issues a redirect for some strange reason.

The solution is to not use ```newFromTitle``` which requires this request context.  I think this context is what screws things up.  We can also easily get the article via ID like this:
```php
$article = \Article::newFromID( $this->commentTitle->getArticleID() );
```